src/components/BluePraragraph.jsx

```diff
@@ -0,0 +1,3 @@
+export default function BlueParagraph() {
+  return <p style={{color:"blue"}}>This is a new paragraph in blue color!</p>
```

Comment:
quiero que al final se escriba viva pasto

Reply:
It is done, please review again. TY.